<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeerToPeer</class>
 <widget class="QMainWindow" name="PeerToPeer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Peer To Peer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QFrame" name="frame_3">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="welcomeLabel">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <property name="text">
                 <string>Welcome to Peer to Peer CommAudio!
Please enter an IP, Port and press Connect to get started.</string>
                </property>
                <property name="scaledContents">
                 <bool>false</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="connectionControls">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>250</height>
                 </size>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="title">
                 <string>Connection Controls</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_10">
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QGroupBox" name="IPControls">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="title">
                    <string>IP Address</string>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_8">
                    <property name="topMargin">
                     <number>5</number>
                    </property>
                    <item>
                     <widget class="QLineEdit" name="lineIPAddress">
                      <property name="text">
                       <string>192.168.0.</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="PortControls">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="title">
                    <string>TCP Port</string>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_9">
                    <property name="topMargin">
                     <number>5</number>
                    </property>
                    <item>
                     <widget class="QLineEdit" name="linePort">
                      <property name="text">
                       <string>7000</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="portControlsUDP">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="title">
                    <string>UDP Port</string>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_12">
                    <property name="topMargin">
                     <number>5</number>
                    </property>
                    <item>
                     <widget class="QLineEdit" name="lineUDPPort">
                      <property name="text">
                       <string>7005</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_9">
                   <item>
                    <widget class="QPushButton" name="buttonConnect">
                     <property name="text">
                      <string>Connect</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="buttonDisconnect">
                     <property name="text">
                      <string>Disconnect</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="userControls" stretch="0,0">
             <property name="spacing">
              <number>10</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetMinimumSize</enum>
             </property>
             <item>
              <widget class="QGroupBox" name="fileTransferControls">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>200</height>
                </size>
               </property>
               <property name="title">
                <string>File Transfer Controls</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_11">
                <item>
                 <widget class="QGroupBox" name="groupBox_3">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>120</height>
                   </size>
                  </property>
                  <property name="title">
                   <string>File Request</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_14">
                   <item>
                    <widget class="QLineEdit" name="lineEdit"/>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButton">
                     <property name="text">
                      <string>Request File</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="SendMicrophone">
                  <property name="text">
                   <string>Send Microphone Audio</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <property name="topMargin">
                <number>30</number>
               </property>
               <item>
                <widget class="QGroupBox" name="audioControlsFrame">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>200</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="title">
                  <string>Local Audio Controls</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <property name="leftMargin">
                   <number>7</number>
                  </property>
                  <property name="topMargin">
                   <number>7</number>
                  </property>
                  <property name="rightMargin">
                   <number>7</number>
                  </property>
                  <property name="bottomMargin">
                   <number>7</number>
                  </property>
                  <item>
                   <layout class="QGridLayout" name="gridLayout_2">
                    <item row="1" column="0">
                     <layout class="QHBoxLayout" name="horizontalLayout_4">
                      <item>
                       <widget class="QPushButton" name="buttonRewindAudio">
                        <property name="text">
                         <string>Rewind</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="buttonStopAudio">
                        <property name="text">
                         <string>Stop</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="buttonForwardAudio">
                        <property name="text">
                         <string>Fast Forward</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item row="0" column="0">
                     <layout class="QHBoxLayout" name="horizontalLayout_2">
                      <item>
                       <widget class="QPushButton" name="buttonPlay">
                        <property name="text">
                         <string>Play</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="buttonPauseAudio">
                        <property name="text">
                         <string>Pause</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="groupBox_5">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>50</height>
                     </size>
                    </property>
                    <property name="title">
                     <string>Sound</string>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_6">
                     <item>
                      <widget class="QSlider" name="sliderSound">
                       <property name="maximum">
                        <number>100</number>
                       </property>
                       <property name="pageStep">
                        <number>1</number>
                       </property>
                       <property name="value">
                        <number>100</number>
                       </property>
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QGroupBox" name="controlsFrame">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>Send Controls</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>Available Music Files</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QListWidget" name="listMusicFiles"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="QueueAddButton">
                 <property name="text">
                  <string>Insert In Queue</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="QueueRemoveButton">
                 <property name="text">
                  <string>Remove From Queue</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="OpenPathButton">
                 <property name="text">
                  <string>Find Files...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>Local Music Queue</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QListWidget" name="listQueueFiles"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="StatusBar">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>728</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
